app/kumactl/cmd/get/get_meshes_test.go

```diff
@@ -76,12 +76,64 @@ var _ = Describe("kumactl get meshes", func() {
 						Path: "/non-standard-path",
 					},
 				},
+				Logging: &v1alpha1.Logging{
+					DefaultBackend: "default-backend",
+					Backends: []*v1alpha1.LoggingBackend{
+						{
+							Name: "backend-1",
+						},
```

Comment:
Can we provide more complete and valid backend here?
Aside from name we should also specify the type of the backend and required fields. Feel free to consult kuma.io as well as try to apply YAML with example via kumactl to trigger validation